baselines/train/configs.py

```diff
@@ -42,13 +42,13 @@
     'commons_harvest__private_property_1',
     'commons_harvest__open_disable_zapping_0',
     'commons_harvest__open_disable_zapping_1',
+    'commons_harvest__open_abundance'
```

Comment:
We add two evaluation scenarios per substrate
```suggestion
    'commons_harvest__open_abundance_0',
    'commons_harvest__open_abundance_1',
```

Reply:
I was meaning to ask what this was about.... what is the difference between 0, and 1 here?

Reply:
Nothing, just evaluating two times ( the same  to verify that the evaluation is not biased by one sample )